examples/games/trex/lib/game_over.dart

```diff
@@ -33,10 +33,10 @@ class GameOverText extends SpriteComponent with HasGameRef<TRexGame> {
   }
 
   @override
-  void onGameResize(Vector2 gameSize) {
```

Comment:
funny how some people implied it was the gameSize, some that is was the canvasSize!

Reply:
And most of the time gameSize == canvasSize 😄